### PR TITLE
bom - button added to update item desc.

### PR DESCRIPTION
**Pull request by neilLasrado**
https://github.com/frappe/erpnext/issues/2585

### CODE REVIEW

File: erpnext/manufacturing/doctype/bom/bom.py [hunk 1]
```diff
@@ -132,6 +132,16 @@ def update_cost(self):
 			self.ignore_validate_update_after_submit = True
 			self.calculate_cost()
 		self.save()
+		
+	def update_item_desc(self):
+		if self.docstatus == 1:
```
Comment (nabinhait) [inline erpnext/manufacturing/doctype/bom/bom.py]: Allow for draft bom as well

File: erpnext/manufacturing/doctype/bom/bom.js [hunk 2]
```diff
@@ -9,6 +9,9 @@ cur_frm.cscript.refresh = function(doc,dt,dn){
 	if (!doc.__islocal && doc.docstatus<2) {
 		cur_frm.add_custom_button(__("Update Cost"), cur_frm.cscript.update_cost,
 			"icon-money", "btn-default");
+		
+		cur_frm.add_custom_button(__("Update Item"), cur_frm.cscript.update_item_desc,
```
Comment (nabinhait) [inline erpnext/manufacturing/doctype/bom/bom.js]: "Update Item Description"?

**Comment by nabinhait** [comment]
Also can you add "Item Name" in Materials and Exploded Materials table?

**Comment by neilLasrado** [comment]
will be done with #2603

**Pull request merged**